README.md

```diff
@@ -1,41 +1,44 @@
 
 # Singapore Tech Internships!
 
-All positions are Software Engineering roles, unless otherwise specified in the Notes column. Similarly, all positions are open to anyone enrolled in a Bachelor's degree program, unless specific graduation years are mentioned in the Notes column. Internships here are for mostly for Summer 2019.
+All positions are for Software Engineering roles and are open to anyone enrolled in a Bachelor's degree program, unless otherwise specified in the Notes column.
 
-**Note**: Some companies do not have a website dedicated for recruitment or internship applications. Some others do not have an internship position listed on their recruitment page although they do accept interns. In these cases, even though the company is listed below, they might not link to a application page.
+**Note:** Internship details listed here are for mostly for Summer 2019, and are accurate as of the time of writing. Some companies do not have a website dedicated for recruitment or internship applications. Some others do not have internship positions listed on their recruitment page although they do accept interns, or may update their openings at a later time.
```

Comment:
I try to keep this guide unopinionated, but I chose to phrase things this way because I can imagine (and from personal experience) that people would read/hear from somewhere that "company xyz only does _______ or doesn't do _______" and subconsciously conclude that those places are no longer viable options, even though that's not necessarily the case.